largestPrimePalindrome/largestPrimePalindrome.js

```diff
@@ -6,3 +6,39 @@
  * Extra credit: Modify the function to look for the largest prime palindrome less than any provided number.
  * Extra credit 2: How can you improve the time complexity?  (Google: Sieve of Eratosthenes)
  */
+// Function
+const largestPP = () => {
+  // Create isPrime function
+  const isPrime = (num) => {
+    let count = 0;
+    for (let i = num; i > 0; i--) {
+      if (num % i === 0) {
+        count++;
+      }
+      if (count > 2) {
+        return false;
+      }
+    }
+    return true;
+  }
+  // For loop starting at 1000 and counting down
+  for (let i = 1000; i > 10; i--) {
```

Comment:
Awesome! To me this shows that you know whats going on. It's similar to the solution that I presented, but it has its own flair to it and you're just using the for loop as you place for doing the heavy lifting. Good stuff man.